src/System.Data.Odbc/src/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netfx-Windows_NT;
+      netstandard;
```

Comment:
Why do we need netstandard here? Wouldn't the rest of the configurations cover supported netstandard on different OS?

Reply:
The problem we hit when we don't validate against netstandard (and instead validating against concrete frameworks like netfx and netcoreapp, etc.) is that then we don't have a "rid-less" build configuration for the src project.

For the "Rid-less" configuration we are now producing a turd assembly that just throws a PlatformNotSupportedException

Reply:
Yep, that explanation is correct. Mono, for example, supports .NET Standard 2.0 but will not select the RID split packages.

Reply:
Thanks